feat: disable html5 validations by default (#2307)
rjsf-team/react-jsonschema-form: docs/3.x upgrade guide.md

From `@babel/preset-env`'s docs

> We leverage [`browserslist`, `compat-table`, and `electron-to-chromium`] to maintain mappings of which version of our supported target environments gained support of a JavaScript syntax or browser feature, as well as a mapping of those syntaxes and features to Babel transform plugins and core-js polyfills.

> ## html5Validate now defaults to false

Reviewer: ```suggestion
## Forms do not enable HTML5 validation by default
```

> In v3.x, `html5Validate` defaults to false. Seem more about [this change here](https://github.com/rjsf-team/react-jsonschema-form/issues/1184).

Reviewer: ```suggestion
In v3.x, the `noHTML5Validate` property has been removed and replaced with `html5Validate`, which has a default value of false. Seem more about [this change here](https://github.com/rjsf-team/react-jsonschema-form/issues/1184).
```

> If you were previously using HTML5 validatins, and want to continue doing so, you can reenable that behavior by setting the prop `noHtml5Validate=false` on your `Form` elements. 

Reviewer: ```suggestion
If you were previously using HTML5 validations, and want to continue doing so, you can reenable that behavior by setting the prop `noHtml5Validate=false` on your `Form` elements. 
```
